cabal-install/tests/UnitTests/Distribution/Client/ProjectConfig.hs

```diff
@@ -696,6 +713,9 @@ instance Arbitrary PackageConfig where
 instance Arbitrary HaddockTarget where
     arbitrary = elements [ForHackage, ForDevelopment]
 
+instance Arbitrary TestShowDetails where
+    arbitrary = elements [Never, Failures, Always, Streaming, Direct]
```

Comment:
`elements [minBound..maxBound]` in case we add/remove some constructors

Reply:
Now that you say it, why not `arbitraryBoundedEnum :: (Bounded a, Enum a) => Gen a`?

Reply:
:+1: